### ISSUE TITLE
Dubbing issue

### ISSUE THREAD

**Issue by cuongnguyen9463** [jianchang512/pyvideotrans#474]
To prevent video and audio from being out of sync when dubbing, I think the dubbing function should be adjusted. After dubbing, combine the small dubbed part with the video cut according to time. Then merge them together.

**Comment by jianchang512** [comment]
Didn't see what it meant, details?

**Comment by cuongnguyen9463** [comment]
Illustrative example:

1
00:00:00,140 --> 00:00:01,730
我穿越到了一个女权的世界 
2
00:00:01,730 --> 00:00:03,080
在这里女人力大无穷 
3
00:00:03,080 --> 00:00:04,190
负责赚钱养家. 

Subtitle translation:

1
00:00:00,140 --> 00:00:01,730
I have transmigrated into a feminist world.
2
00:00:01,730 --> 00:00:03,080
Here, women have boundless strength.
3
00:00:03,080 --> 00:00:04,190
They are responsible for making money to support the family.

Dubbing: Record your voice for the segment 00:00:00,140 --> 00:00:01,730 (I have transmigrated into a feminist world). Then, cut the video into a small segment (00:00:00,140 --> 00:00:01,730) and slow down the video to match the duration of the corresponding voiceover. Create a small video. Then continue with lines 2, 3... until the end.

Combine videos: Finally, combine all the videos created in the previous steps.

It may take longer, but the voiceover will be synchronized with the video.
My English is not good. I use google translate.
